x/feegrant/client/rest/grpc_query_test.go

```diff
@@ -1,3 +1,5 @@
+// +build norace
```

Comment:
I have deleted this file as part of https://github.com/cosmos/cosmos-sdk/pull/9177
So if my PR gets merged first, you'll need to update the test in the file `x/feegrant/keeper/grpc_query_test.go`
If this PR gets merged first, I'll update the `errorMsg` on my side

Reply:
@aleem1314 #9177 got merged, could you resolve the conflicts?